horizons/constants.py

```diff
@@ -500,7 +500,7 @@ class TIER:
 
 	LOWEST = SAILORS
 	HIGHEST = ARISTOCRATS
-	CURRENT_MAX = CITIZENS
+	CURRENT_MAX = MERCHANTS
```

Comment:
After several rounds of trial-and-error, this change causes the `test_settler_level_save_load` test to fail. Will try to investigate further.